plugins/modules/system/shutdown.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: shutdown
+short_description: Shut down a machine
+notes:
+  - C(PATH) is ignored on the remote node when searching for the C(shutdown) command. Use I(search_paths)
+    to specify locations to search if the default paths do not work.
+description:
+    - Shut downs a machine.
+version_added: "1.1.0"
+options:
+  delay:
+    description:
+      - Seconds to wait before shutdown. Passed as a parameter to the shutdown command.
+      - On Linux, macOS and OpenBSD, this is converted to minutes and rounded down. If less than 60, it will be set to 0.
+      - On Solaris and FreeBSD, this will be seconds.
+    type: int
+    default: 0
+  connect_timeout:
+    description:
+      - Maximum seconds to wait for a successful connection to the managed hosts before trying again.
+      - If unspecified, the default setting for the underlying connection plugin is used.
+    type: int
```

Comment:
```suggestion
```
Same here. This was used in `reboot` for the reconnect, if I understand it correctly.

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/700/commits/83b05b476ab4930c1054ed7bb4126bf818a4e0f3.